### ISSUE TITLE
Travis CI expected to fail

### ISSUE THREAD

**Issue by j-mes** [j-mes/j-mes.github.io#21]
Travis CI will continue to fail the tests until all Pa11y accessibility tests pass.

Refer to this once all accessibility issues have been fixed.
